style.css

```diff
@@ -286,3 +286,87 @@ li {
 .lang__tools img {
   margin-right: 42px;
 }
+
+/* Contact-Form */
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url('./images/contact-form-shape.png');
```

Comment:
- [ ] Kindly fix the background image for the entire section of the contact form as per the Figma design.
- Your design:
<img width="367" alt="Screenshot 2022-03-23 at 12 19 47 AM" src="https://user-images.githubusercontent.com/70269507/159579325-ff17bc73-1740-4a85-a43b-b2348728cc16.png">

- Figma design:
<img width="367" alt="Screenshot 2022-03-23 at 12 20 32 AM" src="https://user-images.githubusercontent.com/70269507/159579357-7a97284a-3b4f-4a7f-8eff-0c546fb5fdbe.png">